source/_integrations/xiaomi_ble.markdown

```diff
@@ -63,3 +59,5 @@ Flower Care firmware update steps:
 - Wait for the synchronization of the sensor to finish, and a dialog asking for a firmware update should appear (this might take a few minutes)
 - The installed and latest firmware version can be verified by selecting the plant -> three-dot menu -> Hardware settings -> Hardware update
 - The Flower Care account and app are not required any further for this integration to work
+
+Also note that the battery level of the plant sensor can only be retrieved by connecting to the device (reading characteristics), while the other sensor data is broadcasted passively. To prevent battery drainage, a connection is made only once a day. Connecting to the device also requires that the device has a good signal strength.
```

Comment:
Is there a fixed time of the day for the connection or is totally random ?
What is considered a _good signal strength_ ?

Reply:
There isn't a set time of day.

In my env, -75 was fine.

FWIW, my experience was the battery sensor was a waste of time (inaccurate would be far too generous). This was on multiple devices from different batches running latest firmware.

Reply:
I’m a bit in doubt to really add numbers to a good or bad range. I’m afraid it depends from situation to situation and I want to prevent people complaining if their range is just above and it still won’t work.

Reply:
Indeed. I've seen it work higher, and I've seen it not work lower.

More fun, I've seen the signal strength seemingly change in rooms I'm not in based on the time of day.

Reply:
Ok, I’ll leave the text as it is than.